apps/omg_eth/lib/omg_eth/root_chain.ex

```diff
@@ -52,6 +52,17 @@ defmodule OMG.Eth.RootChain do
     {block_hash, block_timestamp}
   end
 
+  @doc """
+  Returns lists of block submissions from Ethereum logs	
+  """
+  def get_block_submitted_events(from_height, to_height) do
```

Comment:
this is actually a partial revert from https://github.com/omgnetwork/elixir-omg/pull/1376/files#diff-fc70ee7361bc3bb591f254d40b95fcf8L158-L164